### DISCUSSION TITLE
How to set the firstLineChars indent？

### DISCUSSION THREAD

**Discussion by eatkid** [dolanmiu/docx#1619] [Q&A]
now I can set indent by use 
`indent: {
        firstLine: 450
      }`
but how to set indent by chars？
help plz

**Comment by wvbe** [comment]
~You cannot. This is not a missing feature in `docx`, it is simply not a thing in all of OOXML.~

I stand corrected, as @eatkid points out there is the `@w:firstLineChars` attribute. Sorry!

**Reply by eatkid** [reply]
> But you know, this is a very common function in word
> 
> and it is in [ooxml-schemas](https://github.com/dolanmiu/docx/tree/6ede7000fa22fea3706d782bc06e35dcb297eaee/ooxml-schemas)
> 
> <xsd:attribute name="firstLineChars" type="ST_DecimalNumber" use="optional"/>
> 
> So I think it can be achieved, but I don't know what to do

**Comment by eatkid** [comment]
But you know, this is a very common function in word

and it is in
[ooxml-schemas](https://github.com/dolanmiu/docx/tree/6ede7000fa22fea3706d782bc06e35dcb297eaee/ooxml-schemas)

<xsd:attribute name="firstLineChars" type="ST_DecimalNumber" use="optional"/>

So I think it can be achieved, but I don't know what to do
